spk/cops/src/conf/resource

```diff
@@ -3,13 +3,14 @@
     "services": [{
       "service": "cops",
       "display_name": "COPS",
+      "icon": "app/images/cops-{0}.png",
       "type": "apache_php",
       "root": "cops",
       "backend": 2,
       "php": {
         "profile_name": "cops",
         "profile_desc": "PHP profile used by cops",
-        "backend": 6,
+        "backend": 11,
```

Comment:
@smaarn additionally the "zlib" extension is required to view (uncompress) epub books.

Reply:
That's great ! I must confess I never had tried that feature.

This is now fixed.

Reply:
(Thanks for the hint)